src/components/PurchaseOrder/PODetails/PODetailsForm.js

```diff
@@ -198,7 +202,12 @@ class PODetailsForm extends Component {
             xs={6}
             lg={3}
           >
-            <AcqUnits name="acqUnitIds" />
+            <AcqUnitsField
+              name="acqUnitIds"
+              perm={isEditMode ? CREATE_UNITS_PERM : MANAGE_UNITS_PERM}
```

Comment:
if it's edit - use `CREATE_UNITS_PERM`? is it correct?

Reply:
ops, no